mod/nodeinfo.php

```diff
@@ -5,140 +5,141 @@
  * Documentation: http://nodeinfo.diaspora.software/schema.html
 */
 
-require_once("include/plugin.php");
+use \Friendica\Core\Config;
+
+require_once('include/plugin.php');
```

Comment:
Standards: Can you please remove parentheses from systen calls (return, echo, require)?